src/ArcxAnalyticsSdk.ts

```diff
@@ -380,19 +389,31 @@ export class ArcxAnalyticsSdk {
       viewportHeight: window.innerHeight,
       viewportWidth: window.innerWidth,
       url: window.location.href,
+      sessionStorageId: sessionId,
     })
 
     return new ArcxAnalyticsSdk(apiKey, identityId, sdkConfig, websocket)
   }
 
-  private static async getIdentitityId(sdkConfig: SdkConfig, apiKey: string) {
+  private static async _getIdentitityId(sdkConfig: SdkConfig, apiKey: string) {
     const identityId =
       (sdkConfig?.cacheIdentity && window.localStorage.getItem(IDENTITY_KEY)) ||
       (await postRequest(sdkConfig.url, apiKey, '/identify'))
     sdkConfig?.cacheIdentity && window.localStorage.setItem(IDENTITY_KEY, identityId)
     return identityId
   }
 
+  private static _getSessionId() {
+    const existingSessionId = window.sessionStorage.getItem(SESSION_STORAGE_ID_KEY)
+    if (existingSessionId) {
+      return existingSessionId
+    }
+
+    const newSessionId = generateUniqueID()
+    window.sessionStorage.setItem(SESSION_STORAGE_ID_KEY, newSessionId)
+    return newSessionId
+  }
```

Comment:
@tanghyd We talked about the storage yesterday, it actually CAN be persisted between refreshes and page visits:
https://developer.mozilla.org/en-US/docs/Web/API/Window/sessionStorage
> - A page session lasts as long as the tab or the browser is open, and survives over page reloads and restores.